components/molecules/HighlightInput/highlight-input-form.tsx

```diff
@@ -1024,13 +1024,13 @@ const HighlightInputForm = ({ refreshCallback }: HighlightInputFormProps): JSX.E
       )}
 
       <Fab className="md:hidden">
-        <div
+        <button
           onClick={() => setIsFormOpenMobile(true)}
-          className="p-3 mb-10 -mr-4 text-white rounded-full shadow-lg bg-light-orange-10"
+          className="outline-none p-3 mb-10 -mr-4 text-white rounded-full shadow-lg bg-light-orange-10"
           id="mobile-highlight-create"
         >
           <RxPencil1 className="text-3xl" />
-        </div>
+        </button>
       </Fab>
```

Comment:
The `<Fab />` component has button in it, so no need to add a button within it. Also, button text was missing.

```suggestion
      <Fab
        className="md:hidden p-3 mb-10 -mr-4 text-white rounded-full shadow-lg bg-light-orange-10"
        onClick={() => setIsFormOpenMobile(true)}
      >
        <span className="sr-only">Create a highlight</span>
        <RxPencil1 className="text-3xl" />
      </Fab>
```

To make these changes, you will also need to modify the `<Fab />` component in https://github.com/Dun-sin/app/blob/d75dc77058d83949e70b60e7a8f5f7f2ae0b2195/components/atoms/Fab/fab.tsx

```diff
- const Fab = ({ position, children, className }: FabProps) => {
+ const Fab = ({ position, children, className, onClick }: FabProps) => {
```

and 

```diff
  return (
-    <button type="button" className={clsx("fixed", getFabPositionStyles(), className)}>
+    <button type="button" onClick={onClick} className={clsx("fixed", getFabPositionStyles(), className)}> 
      {children}
    </button>
  );
```

Reply:
Thanks for your help with the right direction, Nicky💪🏾